app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -215,6 +215,17 @@ class BrowserToolbarView(
         }
     }
 
+    fun collapse() {
+        // expand only for normal tabs and custom tabs not for PWA or TWA. Mirror expand()
```

Comment:
nit - should comment read "Collapse only"?

Reply:
Thank you!
I'd use lower caps though just to have consistency with the above comment I copied from (and avoiding to change that line also)